db/migrate/20230222155857_add_author_and_post_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorAndPostToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, foreign_key: { to_table: 'users' }
+    add_reference :likes, :post, foreign_key: { to_table: 'posts' }
+  end
+end
```

Comment:
- [ ] This table is not required, please 🙏🏽 remove it. We just need user, post, comments, and likes tables.